.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
   - "0.12"
-  - "4.2"
+  - "4"
```

Comment:
ah, crap. sorry about that. was a bit too eager in trying to follow your lead. I think 4.2 is plenty. I don't really see what having more than one run per Node version is giving us. especially now that major versions are coming out as rapidly as they have been.